### ISSUE TITLE
view - controller

### ISSUE THREAD

**Issue by neebz** [mahnoorfatima/Assignment-with-Express#2]
https://github.com/mahnoorfatima/Assignment-WebScrapper/blob/ec53d357002a0798e84cdf7cbdd3d3e37b3b0ff4/controller.js#L12

`response` and `request` are variable for controller layer. passing them around is a bad design pattern. if you want to pass data to `view` layer, you should explicitly pass them as separate variables in to the method

**Comment by mahnoorfatima** [comment]
Agree. The purpose of sending response to the view, is to check if the response is finished or not.The view (index.js) is behaving as a coding class for view which will control any html file.Will definitely look into advised solution.
